dev/tools/shared/images.json

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "service_name": "agent-sandbox-controller",
+    "context_dir": ".",
+    "dockerfile": "Dockerfile"
+  },
+  {
+    "service_name": "chrome-sandbox",
+    "context_dir": "examples/chrome-sandbox",
+    "dockerfile": "examples/chrome-sandbox/Dockerfile"
+  },
+  {
+    "service_name": "python-runtime-sandbox",
+    "context_dir": "examples/python-runtime-sandbox",
+    "dockerfile": "examples/python-runtime-sandbox/Dockerfile"
+  },
+  {
+    "service_name": "sandbox_router",
+    "context_dir": "clients/python/agentic-sandbox-client/sandbox_router",
+    "dockerfile": "clients/python/agentic-sandbox-client/sandbox_router/Dockerfile"
+  }
+]
```

Comment:
I'm wondering about the long-term maintainability of this approach. How about automatically discovering all Dockerfiles in the repository instead of manually maintaining a list here? My concern is that developers might forget to update this JSON file when adding new images. An auto-discovery mechanism could make the process more seamless and less error-prone.